EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -12608,19 +12608,19 @@ QUEST_LV_0100_20151224_012583	The uncle is said to have shown his respect as soo
 QUEST_LV_0100_20151224_012584	Medzio Diena and the catastrophe involving Orsha have already occurred...{nl}There is one more prophecy that has only been passed on by the bishops of Orsha.{nl}
 QUEST_LV_0100_20151224_012585	You can hear the story in more detail from any of the residents of Orsha, {nl}so why don't you ask around?
 QUEST_LV_0100_20151224_012586	It's an honor to meet the Revelator that I've only heard of through the legends. {nl}You really do seem to be indifferent to the curse as Priest Pranas said...{nl}
-QUEST_LV_0100_20151224_012587	The bishop is thorough... to the point of overcautious.{nl}He had prepared to meet the Revelator in a few more places other than here as well.{nl}
+QUEST_LV_0100_20151224_012587	The bishop is meticulous... to the point of being overly cautious.{nl}He had prepared to meet the Revelator in a few more places other than here as well.{nl}
 QUEST_LV_0100_20151224_012588	Among those numerous places, Ashaq Underground Prison was prepared by myself and Draznie.{nl}
-QUEST_LV_0100_20151224_012589	I was planning on hiding here for a long time...{nl}But since you're here, I think using the prepared items is a better idea.{nl}
+QUEST_LV_0100_20151224_012589	I was planning on hiding here for a long time but...{nl}Since you're here, I think using the prepared items is a better idea.{nl}
 QUEST_LV_0100_20151224_012590	First of all, I've hidden a sack of Pearl Powder in the watchtower.{nl}We can use that against demons to great effect.{nl}
 QUEST_LV_0100_20151224_012591	I'll explain the details once you get back with the Pearl Powder.
 QUEST_LV_0100_20151224_012592	It seems that I still have much to learn...{nl}I can't do anything because of the curse...
 QUEST_LV_0100_20151224_012593	Thank the goddesses, Irma is safe!{nl}Now all we have to do is push the demons out of the Ashaq Underground Prison.
-QUEST_LV_0100_20151224_012594	Once again, the bishop has admired me with his wisdom.{nl}If such preparations hadn't been in place, {nl}we wouldn't have been able to deal with the demons at all.{nl}
-QUEST_LV_0100_20151224_012595	I'd say that the other priest are also in similar hideouts.
+QUEST_LV_0100_20151224_012594	Once again, the bishop has impressed me with his wisdom.{nl}If such preparations hadn't been in place, {nl}we wouldn't have been able to deal with the demons at all.{nl}
+QUEST_LV_0100_20151224_012595	I'd say that the other priests are also in similar hideouts.

```

Comment:
then maybe "my brothers" instead of "other priests" would be better? since i believe that's commonly used in groups like these